README.md

```diff
@@ -296,6 +296,7 @@ List of currently supported languages:
 - [ ] jsdoc
 - [x] dart (maintained by @Akin909)
 - [x] rst (maintained by @stsewd)
+- [x] scheme (maintained by @steelsojka)
```

Comment:
Tree-sitter query language? It's not real scheme.

Reply:
I wasn't sure exactly. What should the official "language" name be? ts-query?

Reply:
I'll rename the parser as well...

Reply:
Does the filetype need to change? `.scm` files are detected as 'scheme' (at least with my setup)

Reply:
Should be enough not make that clear in the README.md. I guess you can keep all other usages of scheme. Until someone wants to parse real scheme... :smile: